src/Icon/Icon.tsx

```diff
@@ -3384,6 +3384,38 @@ export const Icon: FC<Props> = ({
         </svg>
       )
     }
+
+    {render === 'repeat-charge' && (
+      <svg width="100%" height="100%" fill="none" viewBox="0 0 24 24">
+        <path
+          d="M8 16.995c.553 0 1-.453 1-1.005a1 1 0 1 0-2 0V16c0 .552.447.995 1 .995ZM21 3H3C1.346 3 0 4.346 0 6v12c0 1.654 1.346 3 3 3h18c1.654 0 3-1.346 3-3V6c0-1.654-1.346-3-3-3Zm1 15c0 .551-.448 1-1 1H3c-.552 0-1-.449-1-1v-7h20v7Zm0-9H2V6c0-.551.448-1 1-1h18c.552 0 1 .449 1 1v3ZM5 16.995c.553 0 1-.453 1-1.005a1 1 0 1 0-2 0V16c0 .552.447.995 1 .995Z"
+          fill="#112035"
+        />
+        <rect x="12" y="12" width="12" height="12" rx="6" fill="#112035" />
```

Comment:
you might need to update these two fill props with `theme.colours[color]` otherwise it will always be blue

Reply:
Good call!